browserify breaks at require('./lib-cov/mocha')

`mocha/index.js`:

```
module.exports = process.env.COV
  ? require('./lib-cov/mocha')
  : require('./lib/mocha');
```


```
Error: module "./lib-cov/mocha" not found from "/Volumes/work/Projects/yaas/node_modules/mocha/index.js"
```

I'm still unable to browserify a file that require's mocha. Error:
> Error: Cannot find module './lib-cov/mocha' from '/Users/ksikka/test_mochify/node_modules/mocha'

Anyone else in the same boat?

Tried it with node 0.10, 0.12, the latest browserify in npm (11.2.0), the latest mocha in npm (2.3.3).

--- a/package.json
+++ b/package.json
@@ -258,7 +258,6 @@
     "type": "git",
     "url": "git://github.com/mochajs/mocha.git"
   },
-  "main": "./index",
   "bin": {
     "mocha": "./bin/mocha",
     "_mocha": "./bin/_mocha"
@@ -312,7 +311,12 @@
     "debug": "./lib/browser/debug.js",
     "events": "./lib/browser/events.js",
     "tty": "./lib/browser/tty.js",
-    "./index.js": "./browser-entry.js"
+    "./index.js": "./browser-entry.js",
+    "jade": false,
+    "fs": false,
+    "glob": false,
+    "path": false,
+    "supports-color": false
   },
   "licenses": [
     {

